improver_tests/standardise/test_AdjustLandSeaPoints.py

```diff
@@ -63,7 +63,7 @@ def test_basic(self):
                             'output_cube': None}
         result = AdjustLandSeaPoints()
         members = {attr: getattr(result, attr) for attr in dir(result)
-                   if not attr.startswith("__")}
+                   if not attr.startswith("_")}
```

Comment:
Why this change?

Reply:
Because the AdjustLandSeaPoints class now inherits the base class attributes, some of which begin with a single ```_```. So if they are not excluded here, the test will fail. Previously no single underscore attributes existed for this class.